spring-rabbit/src/main/java/org/springframework/amqp/rabbit/listener/SimpleMessageListenerContainer.java

```diff
@@ -770,13 +777,35 @@ private boolean doReceiveAndExecute(BlockingQueueConsumer consumer) throws Throw
 				executeListener(channel, message);
 			}
 			catch (ImmediateAcknowledgeAmqpException e) {
+				if (this.logger.isDebugEnabled()) {
+					this.logger.debug("User requested ack for failed delivery");
```

Comment:
I guess we can't extract `deliveryTag` here to achieve the the same logs like in the `DirectMessageListenerContainer`.
Maybe we can just use the whole `BlockingQueueConsumer.toString()` here?
```
this.logger.debug("User requested ack for failed delivery in: " + consumer);
```